Userland/Utilities/strace.cpp

```diff
@@ -5,19 +5,120 @@
  */
 
 #include <AK/Assertions.h>
+#include <AK/Format.h>
+#include <AK/StdLibExtras.h>
 #include <AK/Types.h>
 #include <LibC/sys/arch/i386/regs.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/File.h>
+#include <errno.h>
+#include <fcntl.h>
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/ptrace.h>
+#include <sys/time.h>
 #include <sys/wait.h>
 #include <syscall.h>
 #include <unistd.h>
 
+#define HANDLE(VALUE) \
+    case VALUE:       \
+        return #VALUE;
+#define VALUES_TO_NAMES(FUNC_NAME)     \
+    static String FUNC_NAME(int value) \
+    {                                  \
+        switch (value) {
+#define END_VALUES_TO_NAMES                \
```

Comment:
Please make this a function-like macro just like the one above, purely for consistency :P

Reply:
Yes, I also thought about making it consistent, it fell through the cracks, will do.

Reply:
Done.